lib/process_settings/monitor.rb

```diff
@@ -31,11 +31,11 @@ def clear_instance
       end
 
       def instance
-        ActiveSupport::Deprecation.warn("ProcessSettings::Monitor.instance is deprecated and will be removed in v1.0. Use ProcessSettings.instance instead.")
         @instance ||= default_instance
       end
 
       def default_instance
+        ActiveSupport::Deprecation.warn("ProcessSettings::Monitor.default_instance is deprecated and will be removed in v1.0. Use ProcessSettings.instance instead.")
```

Comment:
```suggestion
        ActiveSupport::Deprecation.warn("ProcessSettings::Monitor.default_instance is deprecated and will be removed in v1.0. Assign a `FileMonitor` object to `ProcessSettings.instance=` instead.")
```

Reply:
I've gone ahead and applied this change. I think this resolves what we're both trying to do, which is mainly the following:
1. Throw as few deprecation warnings from a single path as possible
2. Make sure that what ever deprecation warnings are raised, they are accurate and actionable.